suite-native/storage/src/typedPersistReducer.ts

```diff
@@ -7,12 +7,14 @@ export const preparePersistReducer = async <TReducerInitialState>({
     reducer,
     persistedKeys,
     key,
+    version,
```

Comment:
I would maybe add a comment in which cases we should bump the version parameter

Reply:
That definitely doesn't belong here because you won't notice it anyway, that must be general knowledge of both us and Suite desktop team because it should be done if you are changing reducer store shape.